weed/command/filer_sync.go

```diff
@@ -90,6 +91,7 @@ func init() {
 	syncMemProfile = cmdFilerSynchronize.Flag.String("memprofile", "", "memory profile output file")
 	syncOptions.metricsHttpIp = cmdFilerSynchronize.Flag.String("metricsIp", "", "metrics listen ip")
 	syncOptions.metricsHttpPort = cmdFilerSynchronize.Flag.Int("metricsPort", 0, "metrics listen port")
+	syncOptions.doDeleteFiles = cmdFilerSynchronize.Flag.Bool("doDeleteFiles", true, "delete and update files when synchronizing")
```

Comment:
The user may need to set for `a` or `b` sync direction.

Reply:
> The user may need to set for `a` or `b` sync direction.

done